model/task_lifecycle.go

```diff
@@ -1017,6 +1022,14 @@ func updateBuildStatus(b *build.Build) (bool, error) {
 	}
 
 	buildStatus, allTasksBlocked := getBuildStatus(buildTasks)
+	if buildStatus == evergreen.BuildUnscheduled {
+		if err = b.SetActivated(false); err != nil {
+			return true, errors.Wrapf(err, "setting build '%s' as inactive", b.Id)
+		}
+
+		return true, nil
```

Comment:
Should this instead be a return value like AllTasksBlocked (could return a small struct to make this neater)? Having it be a status seems misleading since we don't actually use Unscheduled as a build status.

Reply:
Are you referring to getBuildStatus returning evergreen.BuildUnscheduled as a status on line 963?

Reply:
Right; should have commented on 1024 but intsead of checking buildStatus it may make more sense to check something like allTasksUnscheduled, since unscheduled isn't truly a status

Reply:
done